types/underscore/index.d.ts

```diff
@@ -5186,18 +5186,18 @@ declare module _ {
         * Wrapped type `any[]`.
         * @see _.filter
         **/
-        filter(iterator: _.ListIterator<T, boolean>, context?: any): _Chain<T>;
+        filter(iterator: _.ListIterator<T, boolean>, context?: any): _Chain<T, T[]>;
 
         /**
-        * @see _.filter
+        * @see _.select
```

Comment:
actually the original one is better as it is easier to find in the unserscore website using the formal name and not the alias

Reply:
Good point, reverted.